Userland/Utilities/image.cpp

```diff
@@ -240,6 +245,24 @@ static ErrorOr<Gfx::IntRect> parse_rect_string(StringView rect_string)
     return Gfx::IntRect { numbers[0], numbers[1], numbers[2], numbers[3] };
 }
 
+static ErrorOr<unsigned> parse_webp_allowed_transforms_string(StringView string)
+{
+    unsigned allowed_transforms = 0;
+    for (StringView part : string.split_view(',')) {
+        if (part == "predictor" || part == "p")
+            allowed_transforms |= 1;
+        else if (part == "color" || part == "c")
+            allowed_transforms |= 2;
+        else if (part == "subtract-green" || part == "sg")
+            allowed_transforms |= 4;
+        else if (part == "color-indexing" || part == "ci")
+            allowed_transforms |= 8;
```

Comment:
Instead of hardcoding these integers, can you use the existing constants or introduce an enum?

Reply:
Sure, done. Not sure why I didn't do that like that the first time round.